mu/modes/circuitpython.py

```diff
@@ -64,6 +64,8 @@ class CircuitPythonMode(MicroPythonMode):
         (0x04D8, 0xEAD2, None, "DynOSSAT-EDU-OBC"),
         (0x1209, 0x4DDD, None, "ODT CP Sapling M0"),
         (0x1209, 0x4DDE, None, "ODT CP Sapling M0 w/ SPI Flash"),
+        (0x239A, 0x80AC, None, "Unexpected Maker FeatherS2"),
```

Comment:
If this is the Adafruit PID? In that case the board will be automatically detected without this addition.